src/library_html5.js

```diff
@@ -2430,6 +2436,7 @@ var LibraryHTML5 = {
       {{{ makeSetValue('canvas.canvasSharedPtr', 0, 'width', 'i32') }}};
       {{{ makeSetValue('canvas.canvasSharedPtr', 4, 'height', 'i32') }}};
     }
+#endif
 
     if (canvas.offscreenCanvas || !canvas.controlTransferredOffscreen) {
```

Comment:
This line refers to `canvas.offscreenCanvas`.. so why its not also within the `#if OFFSCREENCANVAS_SUPPORT` block?

Reply:
Ah, this probably needs also to be within the `#if OFFSCREENCANVAS_SUPPORT` conditional directive. I've reverted this for now with commit 74468caea8bd90760285bd0019d51c602e429214, and will make another PR with this.